tests/unit/misc/test_sessions.py

```diff
@@ -32,7 +32,11 @@
 from qutebrowser.browser.webkit import tabhistory
 
 
-pytestmark = pytest.mark.qt_log_ignore('QIODevice::read.*: device not open')
+@pytest.fixture(autouse=True)
+@pytest.mark.qt_log_ignore('QIODevice::read.*: device not open')
+def autouse(fake_save_manager):
```

Comment:
What tests is this really needed for? Could you just add `fake_save_manager` to the `sess_man` fixture or so?